lib/elixir_analyzer/constants.ex

```diff
@@ -71,6 +71,9 @@ defmodule ElixirAnalyzer.Constants do
       "elixir.high-score.use_default_argument_with_module_attribute",
     high_score_use_map_update: "elixir.high-score.use_map_update",
 
+    # High School Sweetheart Comments
+    high_school_sweetheart_function_reuse: "elixir.high_school_sweetheart.function_reuse",
```

Comment:
Tests are failing, you need to  change to
```suggestion
    high_school_sweetheart_function_reuse: "elixir.high-school-sweetheart.function_reuse",
```